features/rego/v0/rego_default_v0.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2024 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// Package rego_default_v0 sets the default [ast.RegoVersion] to [ast.RegoV0].
+//
+// Usage (import side effects only):
+//
+//	import _ "github.com/open-policy-agent/opa/features/rego/v0
```

Comment:
```suggestion
//	import _ "github.com/open-policy-agent/opa/features/rego/v0 // default to rego v0
```
I think it's good practice to give a reason for underscore imports; let's give a good example.